lua/nvim-treesitter/highlight.lua

```diff
@@ -177,6 +179,9 @@ function M.set_custom_captures(captures)
 end
 
 function M.set_default_hlgroups()
+  if not ts.highlighter.hl_map then
+    link_all_captures()
```

Comment:
I think this has to run at least once

Reply:
@theHamsta it gets called in the plugin file

https://github.com/nvim-treesitter/nvim-treesitter/blob/549723ff7ba495dbda580fffebb358fbfbda443e/plugin/nvim-treesitter.lua#L44

so it will get called when all the other highlights get set